.github/workflows/kafka-ws.yml

```diff
@@ -0,0 +1,80 @@
+name: Kafka WS (subscription-engine/kafka-ws)
+
+# Triggered after a branch is created, a tag or a branch is pushed or a PR with changes in
+# 'subscription-engine/kafka-ws/**' is submitted
+on:
+  create:
+  push:
+  pull_request:
+    paths:
+      - 'subscription-engine/kafka-ws/**'
+
+jobs:
+  unit-tests:
+    # Path to help identify each job in the status checks
+    name: (subscription-engine/kafka-ws) Unit Tests
+    # If the commit message contains 'ci skip', it is not necessary to run this job
+    if: contains(github.event.head_commit.message, 'ci skip') == false
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
+    steps:
+
+    - name: checkout
+      uses: actions/checkout@v1
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: npm install, build, lint and test
+      run: |
+        cd subscription-engine/kafka-ws
+        npm ci
+        npm run build --if-present
+        npm run parser:compile
+        npm run lint
+        npm test
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v1
+      with:
+        # Keep in mind that secrets are not available in forks of repositories, that's the reason we use an open token
+        token: 6a610aca-9ca1-4ac5-97e1-18eea5f44612
+        # Created when 'npm test' was executed
+        file: subscription-engine/kafka-ws/coverage/lcov.info
+        # Flags allow you to isolate coverage reports for different tests
+        flags: kafka-ws
```

Comment:
I think the flags are just alphanumeric

Reply:
I didn't find the rules for flags names, but the GH Action executed normally in my repository

Reply:
Because you are not uploading yet, but you can change later too